axelrod/tournament.py

```diff
@@ -3,32 +3,64 @@
 import logging
 import multiprocessing
 
-from .game import *
-from .result_set import *
-from .round_robin import *
+from .game import Game
+from .result_set import ResultSet
+from .tournament_type import round_robin
+from .payoff import payoff_matrix
+from .cooperation import cooperation_matrix
 
 
 class Tournament(object):
     game = Game()
 
-    def __init__(self, players, name='axelrod', game=None, turns=200,
+    def __init__(self, players, type=None, name='axelrod',
+                 game=None, turns=200,
                  repetitions=10, processes=None, prebuilt_cache=False,
-                 noise=0, with_morality=True):
+                 noise=0, with_morality=True, keep_matches=False):
+        """
+        Parameters
+        ----------
+        players : list
+            A list of axelrod.Player objects
+        name : string
+            A name for the tournament
+        game : axelrod.Game
+            The game object used to score the tournament
+        turns : integer
+            The number of turns per match
+        repetitions : integer
+            The number of times the round robin should be repeated
+        processes : integer
+            The number of processes to be used for parallel processing
+        prebuilt_cache : boolean
+            Whether a cache has been passed in from an external object
+        noise : float
+            The probability that a player's intended action should be flipped
+        with_morality : boolean
+            Whether morality metrics should be calculated
+        with_interactions : boolean
+            Whether interaction results should be included in the output
+        """
         self.name = name
+        if type is not None:
```

Comment:
Can we avoid using `type` as a variable here since it's a python builtin? (I wish it wasn't a built in.)